modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase

```diff
@@ -244,6 +244,66 @@ definition {
 			viewNoLocationInfo = "Site Name");
 	}
 
+	@description = "This test ensures that a folder can be filtered from the current asset library."
+	@priority = "5"
+	test FilterFolderFromCurrentAL {
```

Comment:
We need to refactor our naming at some point. We have AL, AssetLibrary, AssetLibraries, Depot as names in our Poshi tests. Let's try to use one name for all of them

Reply:
sure, I can file a test improvement ticket to refactor the name later.

Reply:
see https://issues.liferay.com/browse/LRQA-66061

Reply:
Thanks! Yeah let's use Depot whenever possible